### PR TITLE
Check is SQL DBs are Zone Redundant

### PR DESCRIPTION
**Pull request by gunnarcms**

### CODE REVIEW

File: samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+   "properties": {
+     "displayName": "Audit SQL Database is Zone Redundant",
```
Comment (pilor) [inline samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json]: >Audit SQL Database is Zone Redundant [](start = 21, length = 36)

Do you mind changing the display name to "SQL databases should be zone redundant" to conform to newer naming standards?

File: samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+   "properties": {
+     "displayName": "Audit SQL Database is Zone Redundant",     
+     "description": "Will check is the SQL databases are configured for Zone Redundant",
```
Comment (pilor) [inline samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json]: >is the SQL databases are configured for [](start = 32, length = 39)

"Audit if SQL databases are not configured to be zone redundant"

File: samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+     "mode": "All",
+     "metadata": {
+       "category": "SQL"      
+     },
+     "parameters": {      
+      }
+     },
```
Comment (pilor) [inline samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json]: this is not valid JSON

File: samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            "field": "type",
+            "equals": "Microsoft.SQL/servers/databases"
+          },
+          {
+            "field": "name",
+            "notEquals": "master"
+          },
+          {
+            "field": "Microsoft.Sql/servers/databases/zoneRedundant",
+            "equals": "false"
```
Comment (pilor) [inline samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json]: what is the default for this field?  If it is false then you also want to audit when the field does not exist (or audit when it is "notEquals": "true")

Comment (gunnarcms) [reply samples/SQL/audit-sql-database-zone-redundant/azurepolicy.json]: great call ! the default is "zoneRedundant": false , I'll change to inequality, thanks!

**Pull request merged**